frontend/src/v5/helpers/viewpoint.helpers.ts

```diff
@@ -191,12 +192,18 @@ export const toColorAndTransparencyDicts = (overrides: GroupOverride[]): Overrid
 };
 
 export const goToView = async (view: Viewpoint) => {
+	if (isEmpty(view?.state?.colored)  && isEmpty(view?.state?.hidden) && isEmpty(view?.camera) && isEmpty(view?.clippingPlanes)) {
```

Comment:
Given this, I think you can delete line `108` in TicketDetailsCard.component.tsx (`if (isEmpty(defaultView)) return;`)